torch/_export/serde/schema.py

```diff
@@ -8,7 +8,7 @@
 from torch._export.serde.union import _Union
 
 # NOTE: Please update this value if any modifications are made to the schema
-SCHEMA_VERSION = (8, 1)
+SCHEMA_VERSION = (9, 1)
```

Comment:
I think recently some folk in our partner team raised some concern over the IR compatibility. Better to check with them before you're updating the schema.